views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.min.js

```diff
@@ -5,4 +5,4 @@ define("taoQtiItem/portableLib/jquery_2_1_1",[],function(){return function(globa
unction(){this.is("enabled")&&self.playRecording()}.bind(play)),play.on("updatestate",function(){self.player.is("idle")||self.getRecording()&&!self._isAutoPlayingBack?this.enable():this.disable()}.bind(play)),this.controls.play=play),1!==this.config.maxRecords&&!0!==this.config.isReviewMode&&(reset=uiElements.controlFactory({id:"reset",label:this.getControlIcon("reset"),container:this.$controlsContainer}),reset.on("click",function(){this.is("enabled")&&(self.resetRecording(),self.updateResetCount())}.bind(reset)),reset.on("updatestate",function(){1<self.config.maxRecords&&self.config.maxRecords===self._recordsAttempts?this.disable():self.player.is("idle")?this.enable():this.disable()}.bind(reset)),this.controls.reset=reset),self.updateControls()},updateControls:function updateControls(){this._delayCallback||this.countdown&&this.countdown.isDisplayed()||_.invoke(this.controls,"updateState")},destroyControls:function destroyControls(){_.invoke(this.controls,"destroy"),this.controls=null},getControlIcon:function getControlIcon(iconId){return"<svg title=\""+iconId+"\"><use xlink:href=\""+this.iconsFileUrl+"#"+iconId+"\"/></svg>"},id:-1,getTypeIdentifier:function getTypeIdentifier(){return"audioRecordingInteraction"},initialize:function initialize(id,dom,config,assetManager){var self=this;event.addEventMgr(this),this.id=id,this.assetManager=assetManager,this.$container=$(dom),this.render(config),qtiCustomInteractionContext.notifyReady(this),this.on("configChange",function(newConfig){self.render(newConfig)}),html.render(this.$container.find(".prompt"))},setResponse:function setResponse(response){var recording=response.base&&response.base.file;recording&&(this.updateResponse(recording),this.player.loadFromBase64(recording.data,recording.mime))},getResponse:function getResponse(){var response=null;return this.getRecording()&&(response={file:this.getRecording()}),{base:response}},resetResponse:function resetResponse(){this.updateResponse(null)},destroy:function destroy(){var self=this,promises=[];return self.recorder&&self.recorder.is("recording")&&promises.push(self.stopRecording()),self.player&&self.player.is("playing")&&promises.push(self.stopPlayback()),promises.push(self.destroyControls()),self.inputMeter&&promises.push(self.inputMeter.destroy()),self.mediaStimulus&&promises.push(self.mediaStimulus.destroy()),self.player&&promises.push(self.player.unload()),self.recorder&&promises.push(self.recorder.destroy()),promises.push(self.resetResponse()),self._cleanDelayCallback(),Promise.all(promises).then(function(){self.inputMeter=null,self.progressBar=null,self.mediaStimulus=null,self.player=null,self.recorder=null})},setSerializedState:function setSerializedState(state){this.setResponse(state&&state.response||state)},getSerializedState:function getSerializedState(){return this.getResponse()}},qtiCustomInteractionContext.register(audioRecordingInteraction)}),define(["audioRecordingInteraction/runtime/audioRecordingInteraction"],function(PCI){return PCI});
```

Comment:
**issue:** Handlebars is not defined before the compiled template requires it. It will fail at runtime. See for example:
```javascript
define("tpl!audioRecordingInteraction/runtime/js/dialog/tpl/body",["taoQtiItem/portableLib/handlebars_4"],function(hb){
...
});
```